src/box/errcode.h

```diff
@@ -301,7 +301,7 @@ struct errcode_record {
 	/*246 */_(ER_INTERFERING_ELECTIONS,	"Interfering elections started")\
 	/*247 */_(ER_ITERATOR_POSITION,		"Iterator position is invalid") \
 	/*248 */_(ER_DDL_NOT_ALLOWED,		"DDL operations are not allowed: %s") \
-
```

Comment:
I think this empty line was there deliberately. To separate the error list from next code blocks.

Reply:
This was unintended, thanks for noticing — fixed.